x-pack/plugins/fleet/server/routes/epm/handlers.ts

```diff
@@ -183,6 +185,40 @@ export const getDataStreamsHandler: FleetRequestHandler<
   }
 };
 
+export const getDataStreamsStatsHandler: FleetRequestHandler<
+  undefined,
+  TypeOf<typeof GetDataStreamsStatsRequestSchema.query>
+> = async (context, request, response) => {
+  try {
+    const coreContext = await context.core;
+    // Query datastreams as the current user as the Kibana internal user may not have all the required permissions
+    const esClient = coreContext.elasticsearch.client.asCurrentUser;
+
+    const [dataStreams, dataStreamsStats] = await Promise.all([
```

Comment:
Are both ES API calls needed to get the results? Isn't the [data stream stats API](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-stream-stats-api.html#data-stream-stats-api) sufficient?

Reply:
It's not enough because the stats don't include information related to `packages`